### PR TITLE
Stepper: background color to fdfdfd

### LINKED ISSUE #67849: Task: Change Tailored Flows background color from #FFFFFF to #FDFDFD in Stepper Steps
### Details

I think this thread is quite illustrative about what I mean: p1663182275243719-slack-C03E1LGV796

This change is only needed in Stepper, the /start parts are accurate.

### Checklist

_No response_

### Related

_No response_

### PR DESCRIPTION
**Pull request by escapemanuele**
#### Proposed Changes

Change flow background color from #FFFFFF to #FDFDFD
#### Testing Instructions

* Checkout branch and `yarn start`
* Test newsletter and link in bio flow, background color should be #FDFDFD

Related to #67849
Fixes #67849

### CODE REVIEW

**Comment by simison** [comment]
> Test newsletter and link in bio flow, background color should be #FDFDFD

This should affect main onboarding flows too, no? So everything from /start -> Goals -> Vertical question -> Design picking.

**Comment by escapemanuele** [comment]
Well, this should affect all flows that live under Stepper :)

File: client/landing/stepper/declarative-flow/internals/global.scss [hunk 1]
```diff
@@ -12,6 +12,7 @@
  */
 body,
 button {
+	background-color: #fdfdfd;
```
Comment (alshakero) [inline client/landing/stepper/declarative-flow/internals/global.scss]: Are buttons meant to be affected as well?

Comment (escapemanuele) [reply client/landing/stepper/declarative-flow/internals/global.scss]: Obviously not, thanks for catching 😄

**Review by alshakero — APPROVED** [review]
🚢

**Pull request merged**